python_modules/dagster/dagster/core/execution_plan/simple_engine.py

```diff
@@ -42,7 +42,11 @@
 
 
 def start_inprocess_executor(
-    pipeline_context, execution_plan, intermediates_manager, step_keys_to_execute=None
+    pipeline_context,
+    execution_plan,
+    intermediates_manager,
+    step_keys_to_execute=None,
+    emit_skips=True,
```

Comment:
why have a toggle for this? I would wait to add this til we have a case that needs it.